### ISSUE TITLE
"Press [A] to start program" message is stupid

### ISSUE THREAD

**Issue by kervinck** [kervinck/gigatron-rom#38]
Historically [A] refers to the game controller's primary action button. However, the buttons on the game controllers that we have aren't actually labelled, so you have to know that one is [A] and the other is [B].

It gets worse with the new keyboard interface, as the menu doesn't respond to hitting the 'A' key on the keyboard at all. You have to know that 'Del' sends the same code as game controller's [A] button: 0x7f. (that is why 'Del' works). 'End' also works because we mapped it to the same code.

- We could at least make the menu application accept any keyboard key to select the program.... At least 'A' and 'Enter'.
- And regarding the confusing text: why not replace it with a graphic of the game controller.

**Comment by kervinck** [comment]
Fix is just 2 bytes:
```
-    {buttonA}
-    \buttonState, 127^ if<>0loop
+    {buttonA, or any ASCII key from keyboard, 'A' included...}
+    \buttonState, 128& if<>0loop
```

**Comment by kervinck** [comment]
Solved with 5916b628e1e4ba0e2ea055c8a49ee73ec3a583a2

**Comment by kervinck** [comment]
Problem description: https://youtu.be/cwNBfnNO4AY?t=507
